lightning/generate_audio_local/get_speech_lightning_english_api_example.py

```diff
@@ -2,18 +2,23 @@
 from datetime import datetime
 import wave
 
-TOKEN = "ENTER YOUR TOKEN HERE"
+TOKEN = "YOUR_TOKEN_HERE"
 
 ## Enter the URL
 url = "http://waves-api.smallest.ai/api/v1/lightning/get_speech"
 SAMPLE_RATE = 24000 ## Can be changed to 8000, 16000, 48000
-VOICE_ID = "jessica"  ## Other Speakers: arman, james, emily 
+VOICE_ID = "emily"  ## Supported Speakers: ["emily", "jasmine", "arman", "james", "mithali", "aravind", "raj"]
```

Comment:
link to documentation to get the list of all speakers rather than listing them here